src/components/DeFi/components/FormField.tsx

```diff
@@ -0,0 +1,15 @@
+import { Stack, StackProps } from '@chakra-ui/react'
+import { RawText } from 'components/Text'
+
+export type FormFieldProps = {
+  label: string
+} & StackProps
+
+export const FormField: React.FC<FormFieldProps> = ({ label, children, ...rest }) => {
```

Comment:
Non-blocking, but I like to name these `styleProps` to convey intent: You can have OwnProps that are clearly collocated with the component, and any of the additional Chakra UI style props (which in these case are `StackProps`) that you can introspect by going to the definition

Reply:
I don't have an opinion this really, It just gets a little ugly looking when you do `styleProps`. This is really a preference thing. So whichever way we go we'd have to do it on everything to maintain the pattern.